fuel/app/classes/presenter/welcome/hello.php

```diff
@@ -16,6 +16,13 @@ class Presenter_Welcome_Hello extends Presenter
 	 */
 	public function view()
 	{
-		$this->name = $this->request()->param('name', 'World');
+    $hogehogedhoge="fugafugafuga";
+    $hogehoddgehoge="fugafugafuga";
+    $hogdddehogehoge="fugafugafuga";
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* Expected 1 space before "="; 0 found
* Expected 1 space after "="; 0 found
* String "fugafugafuga" does not require double quotes; use single quotes instead